app/views/assignment_swap/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% javascript_include_tag("assignment_swap/validateTwoAssignmentsSwapping") %>
+
+<h1><%= current_office.name %> Kitchen Schedule</h1>
+
+<%= form_for(:assignments, html: { id: "swapping_form" }) do |form|%>
```

Comment:
I typically see dasherized ids and classes in html rather than snake case. Rails does use underscores in some html markup for convenience of interactions in controllers (input names correspond to params) but it makes me a little sad.

Here and `table_header` and `submit_btn` below.